src/app/stack/card-details/component-information/component-snippet/component-snippet.component.html

```diff
@@ -44,7 +44,8 @@ <h4>Details of the dependency: <span class="component-name" [innerText]="compone
                 </div>
                 <div [ngClass]="{'col-xs-12': component?.currentVersion === '-----'}">
                     <p>Recommended Version</p>
-                    <p class="info-head" [innerText]="component?.recommended_version"></p>
+                    <p *ngIf="(tabType == 'private' || tabType == 'public' || tabType == 'compDetails')" class="info-head" [innerText]="component?.recommended_version"></p>
+                    <p *ngIf="!(tabType == 'private' || tabType == 'public' || tabType == 'compDetails')" class="info-head" [innerText]="component?.latestVersion"></p>
```

Comment:
Don't use the double element pattern. Can't this be done like below using `?:` operator? I see this double element for same view everywhere in the preview changes as well.

```suggestion
                    <p *ngIf="(tabType == 'private' || tabType == 'public' || tabType == 'compDetails')" class="info-head" [innerText]="(tabType == 'private' || tabType == 'public' || tabType == 'compDetails') ?component?.recommended_version : component?.latestVersion"></p>
```

Reply:
Okay, i will not use double element pattern. 
Initially i tried with ngif/else but it did not give the expected output..
But yes this ternary approach will definitely work..

Reply:
I have updated your comment ==>
` <p *ngIf="(tabType == 'private' || tabType == 'public' || tabType == 'compDetails')" class="info-head" [innerText]="(tabType == 'private' || tabType == 'public' || tabType == 'compDetails') ?component?.recommended_version : component?.latestVersion"></p>`

to ==>>

 `<p  [innerText]="(tabType == 'private' || tabType == 'public' || tabType == 'compDetails') ? component?.recommended_version : component?.latestVersion"></p>`